freemocap/diagnostics/download_data.py

```diff
@@ -0,0 +1,67 @@
+import io
+import logging
+import zipfile
+from pathlib import Path
+
+import requests
+from dataclasses import dataclass
+from freemocap.system.paths_and_filenames.path_getters import get_recording_session_folder_path
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class Sample:
+    url: str
+
+
+DATASETS = {
+    "test": Sample(
+        url="https://github.com/freemocap/skellysamples/releases/download/test_data_v06_09_25/freemocap_test_data.zip"
```

Comment:
Let's start loading this repo up with sample data, and then use it in the `Data` dropdown menu!

Reply:
https://github.com/freemocap/freemocap/pull/701

being handled in this PR